cnchi/hardware/bumblebee.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+#  bumblebee.py
+#
+#  Copyright © 2013-2016 Antergos
+#
+#  This file is part of Cnchi.
+#
+#  Cnchi is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  Cnchi is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  The following additional terms are in effect as per Section 7 of the license:
+#
+#  The preservation of all legal notices and author attributions in
+#  the material or in the Appropriate Legal Notices displayed
+#  by works containing it is required.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with Cnchi; If not, see <http://www.gnu.org/licenses/>.
+
+
+""" Hybrid bumblebee solution for NVIDIA Optimus Technology
+    Closed source NVIDIA driver & open source intel driver. """
+
+# NVIDIA cards
+CLASSIDS="0300 0302"
+VENDORIDS="10de"
+DEVICEIDS=">/var/lib/mhwd/ids/pci/nvidia.ids"
+
+# Intel cards
+CLASSIDS="0300"
+VENDORIDS="8086"
+DEVICEIDS="*"
+BLACKLISTEDDEVICEIDS="0be1 8108"
+
+try:
+    from hardware.hardware import Hardware
+except ImportError:
+    from hardware import Hardware
+
+import os
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->